### PR TITLE
Move 5 node tests to large tests

### PR DESCRIPTION
**Pull request by ptnapoleon**
@mambocab to review

### CODE REVIEW

**Comment by mambocab** [comment]
So, this could be why we saw the failure in [CASSANDRA-12176](https://issues.apache.org/jira/browse/CASSANDRA-12176)?

**Comment by ptnapoleon** [comment]
Yes, those should be in large tests

File: conf/trunk_test-select.cfg [hunk 1]
```diff
@@ -7,6 +7,9 @@ consistency_test.py:TestAccuracy.test_network_topology_strategy_each_quorum_user
   tools.py:TestAccuracy.test_network_topology_strategy_each_quorum_users
 consistency_test.py:TestAccuracy.test_network_topology_strategy_each_quorum_counters
   tools.py:TestAccuracy.test_network_topology_strategy_each_quorum_counters
+consistency_test.py:TestAccuracy.test_simple_strategy_each_quorum_users
+  tools.py:TestAccuracy.test_simple_strategy_each_quorum_users
+consistency_test.py:TestAccuracy.test_simple_strategy_users
```
Comment (mambocab) [inline conf/trunk_test-select.cfg]: What about `tools.py:...`?

Comment (ptnapoleon) [inline conf/trunk_test-select.cfg]: Don't need it here, since this test isn't annotated with `since`

**Comment by mambocab** [comment]
+1.

**Pull request merged**